md/release-notes.md

```diff
@@ -239,15 +239,45 @@ If you need to use custom types, it is recommended to use Business Object from t
 * BS-19105	Studio Import: unable to recognize git repositories after studio migration
 * BS-19158	Default SAML configuration doesn't work with AD FS
 
+#### Fixes in Bonita 7.8.4 (2019-04-11)
+#### Fixes in Engine component
+* BR-68	BDM REST API response JSON is huge
+* BR-69	SQLServerException error when deleting cases with more then 2100 subprocesses
+* BS-19024	Custom page update sometimes doesn't work unless you restart the platform
+* BS-19284	Cases open and archived tabs show inconsistent Display and Technical Process Name in the process Name column
+* BS-19295	SELECT widget sets the field of the variable holding the initial value to null when the page/form loads if it is shared by another SELECT widget
```

Comment:
some of those bugfixes are not **engine** fixes

Reply:
moving it to UID. Thanks.